webapp/README.md

```diff
@@ -43,7 +43,7 @@ In a terminal, start frontend GUI server using NG :
 
 ```shell
 $ cd src/main/webapp
-$ npm install --force
```

Comment:
Could you explain the differen between those 2 optionsN

Reply:
--legacy-peer-deps is about ignoring peer dependencies version conflicts, while --force is about ignoring a wider range of errors and checks. And here we just want to temporary ignore peer dependencies version checks (until project updated to the new components library)